### DISCUSSION TITLE
Error: Cannot define the root steamapps location

### DISCUSSION THREAD

**Discussion by begin-again** [ebkr/r2modmanPlus#737] [Q&A]
When  clicking launch modded I get the following error.  Do I have an error in one of the locations? The vanilla game launches normally.

```
An error occured whilst searching Steam library locations

Cannot define the root steamapps location
```

## O/S

- Linux Mint 20.2 Cinnamon
- kernel 5.0.7

## Locations

- Browse data folder
  -  `~/.config/r2modmanPlus-local`
- Browse public folder
  - `~/.config/r2modmanPlus-local/Valheim/profiles/todd`
- Change data folder directly
  - `~/.config/r2modmanPlus-local`
- Change steamdirecory
  - `~/.local/share/Steam`
  - Clicking this will force me to select an executable
- Change Valheim directory
  - `~/.steam/steamsteamapps/common/Valheim`
  - Clicking this will force me to select an executable

**Comment by begin-again** [comment]
figured it out. Still doesn't launch the game but I can launch it with bex so all good

**Reply by Errant-4** [reply]
Do you still remember what the trick was? I encountered the same issue
